package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "blogger-backend",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node app.js",
+    "dev": "nodemon app.js",
+    "start:prod": "NODE_ENV=production nodemon app.js"
```

Comment:
tip: we don't run nodemon on production. No need to change now. You can resolve this comment.

Reply:
Got it vaiya.